contributingGuides/TestRail_Navigation_C+.md

```diff
@@ -0,0 +1,30 @@
+# TestRail Navigation and Process for C+
+
+## Context
+C+ are responsible for finding the appropriate location for a new or updated regression test case before the proposed test steps are posted by the Contributor who fixed the bug. This document is intended to give a short overview on how to navigate TestRail and best practices on choosing the correct location for the proposed steps. 
+As a C+ member, you will have view-only access to Expensify's TestRail account in order to complete this task.     
+
+## Navigating TestRail
+- Expensify's TestRail account is broken down into four projects. For the purpose of this task, we will use the project titled `NewDot - New Expensify (New Format)`. Click on this project.  
+- Within the project, you will find six headers that showcase the sub-categories of this project. Here, click on `Test Suites and Cases`.
```

Comment:
Do you like screenshots as much as me?

<img width="772" alt="Screenshot 2023-02-24 at 12 04 30 PM" src="https://user-images.githubusercontent.com/3885503/221150806-5010996c-a813-489e-859c-66cf4a53b39d.png">

```suggestion
- Within the project, you will find six headers that showcase the sub-categories of this project. Here, click on `Test Suites and Cases`.
    - <img width="772" alt="Screenshot 2023-02-24 at 12 04 30 PM" src="https://user-images.githubusercontent.com/3885503/221150806-5010996c-a813-489e-859c-66cf4a53b39d.png">
```